### ISSUE TITLE
Do commands putting plans in the console?

### ISSUE THREAD

**Issue by Temka193** [SpaceMadness/lunar-unity-console#27]
I would like to add in the console to enter commands.
Is it in your plans Asset?

**Comment by weeeBox** [comment]
Yes, it's in a roadmap: stay tuned and make sure to rate the plugin on the asset store is you find it useful

**Comment by weeeBox** [comment]
Here's the first [step](https://github.com/SpaceMadness/lunar-unity-console/wiki/Actions-and-Variables). Available starting v0.8.0.
